Description: Hack to include CPPFLAGS with qmake compilation
Author: Carlos Henrique Lima Melara <charlesmelara@outlook.com>
Bug: https://github.com/MasterQ32/kristall/issues/244
Applied-Upstream: 0.4, https://github.com/MasterQ32/kristall/commit/be009e47529ea4118079928d7e7f3d3b4123c472
Last-Update: 2022-07-24

Index: kristall/src/kristall.pro
===================================================================
--- kristall.orig/src/kristall.pro
+++ kristall/src/kristall.pro
@@ -28,6 +28,9 @@ linux {
         QMAKE_CFLAGS   *= $(shell dpkg-buildflags --get CFLAGS)
         QMAKE_CXXFLAGS *= $(shell dpkg-buildflags --get CXXFLAGS)
         QMAKE_LFLAGS   *= $(shell dpkg-buildflags --get LDFLAGS)
+        # qmake does not use QMAKE_CPPFLAGS, this is a hack to include it
+        QMAKE_CXXFLAGS += $(shell dpkg-buildflags --get CPPFLAGS)
+        QMAKE_CFLAGS   += $(shell dpkg-buildflags --get CPPFLAGS)
     }
 }
 
